server/core/runtime/policy_check_step_runner.go

```diff
@@ -2,25 +2,38 @@ package runtime
 
 import (
 	"context"
+	"github.com/runatlantis/atlantis/server/lyft/feature"
 
 	"github.com/hashicorp/go-version"
 	"github.com/pkg/errors"
 	"github.com/runatlantis/atlantis/server/events/command"
 )
 
+type executor interface {
```

Comment:
Isn't this the same interface as LegacyExecutor? Why not just use the same one across both?

Reply:
it's slightly different, legacy passes both the cmd ctx and global ctx while we just use the cmd ctx

Reply:
It would be less changes overall to just use the same interface and just use `_` for the unused args imo.